package.json

```diff
@@ -4,21 +4,10 @@
   "description": "The official JavaScript / NodeJS SDK for ORY Hydra.",
   "license": "Apache-2.0",
   "main": "sdk/js/swagger/src/index.js",
-  "scripts": {
-    "postinstall": "opencollective postinstall"
-  },
   "browser": {
     "fs": false
   },
   "dependencies": {
     "superagent": "3.7.0"
-  },
```

Comment:
Could you move the open collective stuff into a different package.json, e.g. `package.oc.json`? It's relevant for generating the badges in the readme. Thank you! :)

Reply:
Sure, updated :)